src/_static/css/cylc.css

```diff
@@ -0,0 +1,141 @@
+.logo {
+    height: 110px!important;
+    /* Read the docs theme can't handle vector medium.
+     * Image will not overflow width, so just give a sensible height value. */
+}
+
+/* Correct the behaviour of the read the docs theme which greedily overides
+ * sphinx's `pygments_style` option by placing its own syntax in the
+ * stylesheet - https://github.com/rtfd/sphinx_rtd_theme/issues/166 */
+.highlight * {
+    color: black;
+    font-weight: normal;
+    font-style: normal;
+    background-color: transparent;
+}
+
+/* Correct the behaviour of nested admonitions. Sometimes the admonition-header
+ * inherits its parents styling. */
+.rst-content .error .admonition-title, .rst-content .danger .admonition-title {
+    background: red
+}
+.rst-content .warning .admonition-title,
+.rst-content .caution .admonition-title,
+.rst-content .attention .admonition-title {
+    background: orange
+}
+
+/* Enable theme handling of pilcrow symbols for code-block captions and
+ * admonitions. */
+.rst-content p.admonition-title .pull-left.headerlink,
+.rst-content div.code-block-caption .pull-left.headerlink {
+    margin-right:.3em
+}
+.rst-content p.admonition-title .pull-right.headerlink,
+.rst-content div.code-block-caption .pull-right.headerlink {
+    margin-left:.3em
+}
+.rst-content p.admonition-title .headerlink:before,
+.rst-content div.code-block-caption .headerlink:before {
+    font-family:"FontAwesome";
+    display:inline-block;
+    font-style:normal;
+    font-weight:normal;
+    line-height:1;
+    text-decoration:inherit;
+    font-size:14px;
+    vertical-align:-15%
+    -webkit-font-smoothing:antialiased
+}
+.rst-content p.admonition-title .headerlink:after,
+.rst-content div.code-block-caption .headerlink:after {
+    visibility:visible;
+    content:"";
+    font-family:FontAwesome;
+    display:inline-block;
+    color:#2980B9
+}
+.rst-content p.admonition-title:hover .headerlink,
+.rst-content div.code-block-caption:hover .headerlink {
+    display:inline-block
+}
+.rst-content p.admonition-title .headerlink,
+.rst-content div.code-block-caption .headerlink {
+    font:normal normal normal 14px/1 FontAwesome;
+    text-rendering:auto;
+    -webkit-font-smoothing:antialiased;
+    -moz-osx-font-smoothing:grayscale;
+    display:none;
+    visibility:hidden;
+    font-size:14px;
```

Comment:
Hmm, isn't this one already defined in `font` entry a few lines above? Would we have to update it in two places, or maybe one would be redundant?